.github/actions/cleanup-pe/cleanPE.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/actions/cloud"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/config"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/utils"
+)
+
+func main() {
+	cleanOnlyTaggedPE := false
+	cleanAllEnv := os.Getenv("CLEAN_TAGGED_PE")
+	switch cleanAllEnv {
+	case "true":
+		cleanOnlyTaggedPE = true
+	case "false":
+	default:
+		log.Fatal("CLEAN_TAGGED_PE must be set to true or false")
+	}
+	err := SetGCPCredentials()
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Should we interrupt execution after a fatal error occurs ?

Reply:
I think so. The program will stop if it doesn't get the credentials or if it receives an error from another function. Log.Fatal is equivalent to Print() followed by a call to os.Exit(1).

Reply:
Oh I wasn't aware that the logging library was taking care of interrupting the execution, all good then !